vim_binding.js

```diff
@@ -306,8 +308,8 @@ define([
       'z,t': 'jupyter-notebook:scroll-cell-top',
       'ctrl-j': 'jupyter-notebook:select-next-cell',
       'ctrl-k': 'jupyter-notebook:select-previous-cell',
-      'shift-j': 'jupyter-notebook:extend-marked-cells-below',
-      'shift-k': 'jupyter-notebook:extend-marked-cells-above',
+      'shift-j': 'jupyter-notebook:extend-selection-below',
+      'shift-k': 'jupyter-notebook:extend-selection-above',
```

Comment:
They change the name again....
https://github.com/jupyter/notebook/commit/f05dfae0f3b2368bead6e374779ac21f7948e245#diff-52f8e2b09a62fa92d0a7142010ec4a7bL182